CONTRIBUTION.md

```diff
@@ -71,6 +71,9 @@ git tools.
   - Format your code and fix all the spelling/grammatical mistakes.
   - Limit the column length of the code and the comments within the code to 120 characters.
   - Error messages within the code should be limited to a single line.
+  - Document the details of your PR and put a link inside your PR comment
```

Comment:
Every PR needn't require documentation. Please replace with `Update the documentation if your PR is introducing or changing user facing functionality.`